homu/server.py

```diff
@@ -117,7 +117,7 @@ def result(repo_label, pull):
 
 @get('/queue/<repo_label:path>')
 def queue(repo_label):
-    if repo_label not in g.cfg['repo']:
+    if (repo_label not in g.cfg['repo']) and (repo_label != 'all'):
```

Comment:
I think the parenthesis could be removed.

Reply:
I am not very familiar with the operator precedence in python, but after looking it up I am confident that you are correct and have removed the unnecessary parentheses